tools/cell_census_builder/tests/test_builder.py

```diff
@@ -1,26 +1,186 @@
-# General unit tests for cell_census_builder. Intention is to add more fine-grained tests for builder.
 import os
 from tempfile import TemporaryDirectory
+from typing import List
+from unittest.mock import patch
```

Comment:
No need to change, but I'm curious why you used `unittest.mock` instead of `pytest.monkeypatch`?

It isn't worth changing - we should just make sure we use a consistent style for this (and I don't have a strong opinion on this as pytest and unittest.mock play fine together)

Reply:
it isn't a drop-in replacement.  Lots of people prefer doing what you did as `pytest.monkeypatch` is kind of weird.

To be clear: I think your choice was fine!  I just think we should all treat that as our informal standard for future tests so we have consistency.